nixos/modules/programs/neovim.nix

```diff
@@ -169,9 +169,9 @@ in {
       inherit (cfg) viAlias vimAlias withPython3 withNodeJs withRuby;
       configure = cfg.configure // {
 
-        customRC = (cfg.configure.customRC or "") + ''
-          set runtimepath^=${runtime}/etc
-        '';
+        customRC = ''
+          set runtimepath^=${runtime}
+        '' + (cfg.configure.customRC or "");
```

Comment:
What's an example scenario in your config that requires this re-ordering? On the face of it, this seems like a reasonable change, but I've done a fair amount of vim/neovim configuring and don't recall this being an issue. I wonder if your use-case could be solved by using the "after" directories instead?

Reply:
> I put 99% of my configuration into a [plugin](https://github.com/ivanbrennan/nixbox/tree/88edbaf5d4587e493b803911171ede9891e94ac4/overlays/vim/ncore-plugin)

This sounds a little strange and doesn't work for me. I use lazy.nvim which takes over the plugin loading and I need to config lazy.nvim when it's loaded. I can add my config to rtp manully but that's also strange.

Reply:
in general we tend to setup the rtp before running any custom code, we even wrap nvim with `-u` commands to do that. So setting the rtp earlier sounds good. Even better would be to move the runtime in a location read by default by nvim. See `:help 'runtime'` for a list. For instance writing the runtime to /etc/xdg/nvim it would be picked up automatically.

Reply:
> For instance writing the runtime to /etc/xdg/nvim it would be picked up automatically.

I can implement this. Should I do it in this MR?

Reply:
yes

Reply:
Done.

Reply:
@teto Kindly ping.